<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>calculatrice</class>
 <widget class="QMainWindow" name="calculatrice">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>999</width>
    <height>671</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="ligneCommande">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>531</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto Light</family>
      <pointsize>13</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>295</width>
      <height>301</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <property name="verticalSpacing">
      <number>17</number>
     </property>
     <item row="4" column="1">
      <widget class="QPushButton" name="pushButton_FRAC">
       <property name="text">
        <string>/</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="pushButton_1">
       <property name="font">
        <font>
         <pointsize>7</pointsize>
        </font>
       </property>
       <property name="text">
        <string>1</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="pushButton_LP">
       <property name="text">
        <string>(</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QPushButton" name="pushButton_RP">
       <property name="text">
        <string>)</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="pushButton_9">
       <property name="text">
        <string>9</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QPushButton" name="pushButton_DOT">
       <property name="text">
        <string>.</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="pushButton_7">
       <property name="text">
        <string>7</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="pushButton_3">
       <property name="text">
        <string>3</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>2</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="pushButton_4">
       <property name="text">
        <string>4</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="pushButton_5">
       <property name="text">
        <string>5</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="pushButton_6">
       <property name="text">
        <string>6</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="pushButton_0">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="pushButton_8">
       <property name="text">
        <string>8</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QPushButton" name="pushButton_LB">
       <property name="text">
        <string>[</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QPushButton" name="pushButton_COMPLEXE">
       <property name="text">
        <string>i</string>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QPushButton" name="pushButton_RB">
       <property name="text">
        <string>]</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QPushButton" name="pushButton_SPACE">
       <property name="text">
        <string>ESPACE</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>60</y>
      <width>231</width>
      <height>301</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <property name="horizontalSpacing">
      <number>0</number>
     </property>
     <property name="verticalSpacing">
      <number>7</number>
     </property>
     <item row="2" column="0">
      <widget class="QPushButton" name="pushButton_MUL">
       <property name="text">
        <string>*</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="pushButton_DIVIDE">
       <property name="text">
        <string>รท</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QPushButton" name="pushButton_DOLLAR">
       <property name="text">
        <string>$</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QPushButton" name="pushButton_EQ">
       <property name="text">
        <string>=</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="pushButton_ADD">
       <property name="text">
        <string>+</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QPushButton" name="pushButton_DEN">
       <property name="text">
        <string>DEN</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="pushButton_SOUS">
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="pushButton_MOD">
       <property name="text">
        <string>MOD</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="pushButton_NEG">
       <property name="text">
        <string>NEG</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="pushButton_DIV">
       <property name="text">
        <string>DIV</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QPushButton" name="pushButton_IF">
       <property name="text">
        <string>IFT</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="pushButton_NUM">
       <property name="text">
        <string>NUM</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QPushButton" name="pushButton_RE">
       <property name="text">
        <string>RE</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QPushButton" name="pushButton_IM">
       <property name="text">
        <string>IM</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>360</y>
      <width>531</width>
      <height>141</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0">
      <widget class="QPushButton" name="pushButton_DUP">
       <property name="text">
        <string>DUP</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="pushButton_CLEAR">
       <property name="text">
        <string>CLEAR</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="pushButton_LASTOP">
       <property name="text">
        <string>LASTOP</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="pushButton_LASTARG">
       <property name="text">
        <string>LASTARG</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="pushButton_UNDO">
       <property name="text">
        <string>UNDO</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="pushButton_REDO">
       <property name="text">
        <string>REDO</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="pushButton_DROP">
       <property name="text">
        <string>DROP</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="pushButton_SWAP">
       <property name="text">
        <string>SWAP</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QListWidget" name="affPile">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>100</y>
      <width>201</width>
      <height>451</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto Light</family>
      <pointsize>20</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>510</y>
      <width>531</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="0" column="1">
      <widget class="QPushButton" name="pushButton_FORGET">
       <property name="text">
        <string>FORGET</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="pushButton_STO">
       <property name="text">
        <string>STO</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="pushButton_EVAL">
       <property name="text">
        <string>EVAL</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>50</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>                Mรฉmoire</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>50</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>           Variables</string>
    </property>
   </widget>
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>100</y>
      <width>201</width>
      <height>451</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="affErreur">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>570</y>
      <width>961</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto Light</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">affErreur{
color:rgb(255, 0, 0);
}</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_VALIDER">
    <property name="geometry">
     <rect>
      <x>562</x>
      <y>7</y>
      <width>421</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>V A L I D E R</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>999</width>
     <height>26</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Rounded Elegance</family>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionQuitter"/>
   </widget>
   <addaction name="menuMenu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionQuitter">
   <property name="text">
    <string>Quitter</string>
   </property>
  </action>
  <action name="actionAnnuler_Ctrl_Z">
   <property name="text">
    <string>Annuler ( Ctrl + Z)</string>
   </property>
  </action>
  <action name="actionR_tablie_Ctrl_Y">
   <property name="text">
    <string>Rรฉtablie (Ctrl + Y)</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
